.travis.yml

```diff
@@ -223,11 +223,6 @@ before_install:
      # Check CC variable
     - echo "CC="$CC
 
-    # Write configuration items to standard location to make sure they are
```

Comment:
I'm not completely sure why, but with the other changes that ensure the configuration never needs to be reloaded, this writing of a configuration file now causes failures, so I removed it.

Reply:
This was to fix a problem with parallel tests. :(

#8727

Reply:
What about... Instead of removing this, we add a line to the `astropy.cfg` here to set `auto_download = False`?

Reply:
I wondered about it, but it seemed to me that your solution of a change in `astropy/conftest.cfg` was really more elegant, and works for everybody.

Do you think we still need this at all? It doesn't seem to give failures, and it is hard to see how we can mess up the configuration on travis.

Reply:
Oh, I didn't realize I made it unnecessary. I'll have to remember what I did and get back to you.

Reply:
I am curious... You said this command made the tests fail in Travis CI. However, the same command was also added in #8727 to CircleCI and that didn't affect your PR. How could that be? 🤔

Reply:
I'm totally confused!  But with this still in place (one but last commit), both Initial tests on travis failed (see https://travis-ci.org/github/astropy/astropy/builds/696611940). I'm actually not quite sure why they failed, but trying to determine why the tests passed locally I realized that the only difference was the config file (and locally tests also fail if I use the now-removed lines as config).

Reply:
Given that it still exists in CircleCI for "32 bit and parallel" tests and things pass there, probably okay to remove here. If @astrofrog disagrees when he is available again, we can always do a follow-up PR.